css/types/basic-shape.json

```diff
@@ -294,19 +294,6 @@
                   "partial_implementation": true,
                   "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties."
                 },
-                {
```

Comment:
I was going to say that leaving a gap in the data isn't allowed by the guideline, but it ain't so! The guideline says that we shouldn't leave an _ambiguous_ gap  where the remaining data is non-real. Cool cool.